### ISSUE TITLE
Carnet de visite numérique

### ISSUE THREAD

**Issue by Julie-JDS** [Jardin-des-Sciences/website#15]
Peut-on proposer un carnet de visite numérique qui pourrait être complété 
en ligne ou être imprimé ensuite pour la classe ? 
Est ce que c'est très compliqué et chronophage ?
A discuter en direct pour préciser.

Ex : http://www.tela-botanica.org/page:cel

**Comment by bzg** [comment]
Je pense que ce serait une excellente fonctionnalité, et qu'il faut la prévoir tout de suite, tant que la question du profil utilisateur n'est pas close.

Je vois deux choses:

- une liste de suivi, qui permettrait de suivre les contributions qu'on a soi-même faites

- un carnet de visite, qui permettrait de mettre de côté des dossiers, billets ou éléments qu'on veut visiter plus tard

Je ne pense pas par contre qu'il faille s'engouffrer dans des fonctionnalités de partage de ces listes de suivi ou de carnets : l'idée est tentante, mais risque de ne pas être du tout mise en oeuvre par les utilisateurs.  A voir en fonction des premiers retours sur les usages de la plate-forme.

**Comment by Julie-JDS** [comment]
Ce carnet de visite pourrait être quelque chose de l'ordre de ce que propose 
http://www.tela-botanica.org/page:cel 
ou encore un carnet de visite virtuel, par exemple pour des classes (mais pas que) 
venant visiter tel ou tel musées / activités proposées par le JDS : 
http://web.iri.centrepompidou.fr/fonds/upload/seance/11/Seminaire_Museologie4_R.Topalian.pdf

**Comment by bzg** [comment]
Oui, j'avais bien regardé le premier exemple et le deuxième est dans le registre de la personnalisation.

Que penses-tu de la distinction des deux fonctionnalités ?

Comment vois-tu la mise en oeuvre de chacune ?

**Comment by Julie-JDS** [comment]
J'avoue que c'est un point à préciser et discuter... Je vois bien la différence d'usage, mais je ne mesure pas forcément la différence de complexité de mise en oeuvre. Ca me semble pour les 2 pistes, de "gros projets dans le projet". En même temps, même si ça devait être envisagé plus tard, il est nécessaire d'au moins les considérer comme des perspectives à intégrer dès maintenant pour ne pas être bloqués ensuite... Une bonne réponse de normand dis donc ;)

**Comment by bzg** [comment]
Oui :)

En tous cas j'ai bien noté.
